aws/app/lambda/nagware/nagware.js

```diff
@@ -0,0 +1,58 @@
+const { retrieveFormResponsesOver28DaysOld } = require("dynamodbDataLayer");
+const { getFormNameAndOwnerEmailAddress } = require("postgreSQLDataLayer");
+const { notifyFormsTeam, reportError } = require("slackNotification");
+const { notifyFormOwner } = require("emailNotification");
+
+const ENABLED_IN_STAGING = true;
+
+exports.handler = async (event) => {
+  try {
+    if (!ENABLED_IN_STAGING && process.env.ENVIRONMENT === "staging") return;
+
+    const oldestFormResponseByFormID = await findOldestFormResponseByFormID();
+    await nag(oldestFormResponseByFormID);
+
+    return {
+      statusCode: "SUCCESS",
+    };
+  } catch (error) {
+    await reportError(error.message);
+
+    return {
+      statusCode: "ERROR",
+      error: error.message,
+    };
+  }
+};
+
+async function findOldestFormResponseByFormID() {
+  const unsavedFormResponses = await retrieveFormResponsesOver28DaysOld("New");
+  const unconfirmedFormResponses = await retrieveFormResponsesOver28DaysOld("Downloaded");
+
+  const reduceResult = unsavedFormResponses.concat(unconfirmedFormResponses).reduce((acc, curr) => {
```

Comment:
Ok for now but I'm going to try to see if we can do this in a more efficient way.